intake/source/cache.py

```diff
@@ -240,6 +249,119 @@ def _download(file_in, file_out, blocksize, output=False):
         display.remove(out)
 
 
+class FileCache(BaseCache):
+    """Cache specific set of files
+
+    Input is a single file URL, URL with glob characters or list of URLs. Output
+    is a specific set of local files.
+    """
+
```

Comment:
Should `BaseCache._load` go here too?

Reply:
I don't think so - subclasses may want to call this basic download-a-file, as both Dir and Compressed do.

Reply:
Okay, I didn't realize that.  thanks